Sources/Pandora/Pandora.m

```diff
@@ -296,7 +296,7 @@ - (BOOL) doPartnerLogin: (SyncCallback) callback {
                         @"password":    self.device[kPandoraDevicePassword],
                         @"deviceModel": self.device[kPandoraDeviceDeviceID],
                         @"version":     PANDORA_API_VERSION,
-                        @"includeUrls": [NSNumber numberWithBool:TRUE]
```

Comment:
Be careful doing this — this is not an exact equivalence and it broke Pandora One login.  TRUE doesn't turn into a NSNumber bool whereas YES does.

See the output of the below:

```
  NSData *data;

  data = [NSPropertyListSerialization dataWithPropertyList:@{@"includeUrls": @TRUE} format:NSPropertyListXMLFormat_v1_0 options:0 error:NULL];
  NSLog(@"%@", [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding]);

  data = [NSPropertyListSerialization dataWithPropertyList:@{@"includeUrls": @YES} format:NSPropertyListXMLFormat_v1_0 options:0 error:NULL];
  NSLog(@"%@", [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding]);

  data = [NSPropertyListSerialization dataWithPropertyList:@{@"includeUrls": [NSNumber numberWithBool:TRUE]} format:NSPropertyListXMLFormat_v1_0 options:0 error:NULL];
  NSLog(@"%@", [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding]);
```